pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -134,6 +134,7 @@ let
     "__darwinAllowLocalNetworking"
     "__impureHostDeps" "__propagatedImpureHostDeps"
     "sandboxProfile" "propagatedSandboxProfile"
+    "enabledHardeningOptions"
```

Comment:
This is pretty nasty because it removes each `enabledHardeningOptions` set in a call to `stdenv.mkDerivation`.
Any suggestions on how to improve that?